### ISSUE TITLE
az staticwebapp update Operation returned an invalid status 'Bad Request'

### ISSUE THREAD

**Issue by cocoojbs** [Azure/azure-cli#24387]
## Describe the bug

**Command Name**
`az staticwebapp update`

**Errors:**
```
Operation returned an invalid status 'Bad Request'

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az staticwebapp update -n {} -b {} -s {} -t {}`

## Environment Summary
```
Linux-5.4.0-1094-azure-x86_64-with-glibc2.35 (Cloud Shell), CBL-Mariner/Linux
Python 3.9.13
Installer: RPM

azure-cli 2.41.0

Extensions:
ai-examples 0.2.5
ml 2.10.0
ssh 1.1.2

Dependencies:
msal 1.20.0b1
azure-mgmt-resource 21.1.0b1
```

**Comment by yonzhan** [comment]
route to CXP team

**Comment by PramodValavala-MSFT** [comment]
@cocoojbs Could you share the full command (removing any personal information) and also the complete output when run using the `--debug` flag to provide more details here?

**Comment by cocoojbs** [comment]
@PramodValavala-MSFT 
I ran the code with the --debug option given and got the following debug log. This time I see that the repositoryUrl for staticwebapp has changed. However, the reference to the production environment does not appear to have changed. Is this correct behavior?

full command:
az staticwebapp update -n {staticwebappNAME} -b {branchNAME} -s {NEW-github-sourceURL} -t {TOKEN}

----- run using the --debug flag -----
cli.knack.cli: __init__ debug log:
Enable color in terminal.
cli.knack.cli: Event: Cli.PreExecute []
cli.knack.cli: Event: CommandParser.OnGlobalArgumentsCreate [<function CLILogging.on_global_arguments at 0x7f1b23aed160>, <function OutputProducer.on_global_arguments at 0x7f1b239ad4c0>, <function CLIQuery.on_global_arguments at 0x7f1b239b2a60>]
cli.knack.cli: Event: CommandInvoker.OnPreCommandTableCreate []
cli.azure.cli.core: Modules found from index for 'staticwebapp': ['azure.cli.command_modules.appservice']
cli.azure.cli.core: Loading command modules:
cli.azure.cli.core: Name                  Load Time    Groups  Commands
cli.azure.cli.core: appservice                0.167        72       251
cli.azure.cli.core: Total (1)                 0.167        72       251
cli.azure.cli.core: These extensions are not installed and will be skipped: ['azext_next']
cli.azure.cli.core: Loading extensions:
cli.azure.cli.core: Name                  Load Time    Groups  Commands  Directory
cli.azure.cli.core: ai-examples               0.118         1         1  /usr/lib/python3.9/site-packages/azure-cli-extensions/ai-examples
cli.azure.cli.core: Total (1)                 0.118         1         1  
cli.azure.cli.core: Loaded 73 groups, 252 commands.
cli.azure.cli.core: Found a match in the command table.
cli.azure.cli.core: Raw command  : staticwebapp update
cli.azure.cli.core: Command table: staticwebapp update
cli.knack.cli: Event: CommandInvoker.OnPreCommandTableTruncate [<function AzCliLogging.init_command_file_logging at 0x7f1b22c25b80>]
cli.azure.cli.core.azlogging: metadata file logging enabled - writing logs to '/home/kitamurat/.azure/commands/2022-10-26.01-11-22.staticwebapp_update.252.log'.
az_command_data_logger: command args: staticwebapp update -n {} -b {} -s {} -t {} --debug
cli.knack.cli: Event: CommandInvoker.OnPreArgumentLoad [<function register_global_subscription_argument.<locals>.add_subscription_parameter at 0x7f1b22c4f790>]
cli.knack.cli: Event: CommandInvoker.OnPostArgumentLoad []
cli.knack.cli: Event: CommandInvoker.OnPostCommandTableCreate [<function register_ids_argument.<locals>.add_ids_arguments at 0x7f1b22b91f70>, <function register_cache_arguments.<locals>.add_cache_arguments at 0x7f1b22ba10d0>]
cli.knack.cli: Event: CommandInvoker.OnCommandTableLoaded []
cli.knack.cli: Event: CommandInvoker.OnPreParseArgs []
cli.knack.cli: Event: CommandInvoker.OnPostParseArgs [<function OutputProducer.handle_output_argument at 0x7f1b239ad550>, <function CLIQuery.handle_query_parameter at 0x7f1b239b2af0>, <function register_ids_argument.<locals>.parse_ids_arguments at 0x7f1b22ba1040>]
cli.azure.cli.core.commands.client_factory: Getting management service client client_type=WebSiteManagementClient
urllib3.connectionpool: Starting new HTTP connection (1): localhost:50342
urllib3.connectionpool: http://localhost:50342 "POST /oauth2/token HTTP/1.1" 200 3390
msrestazure.azure_active_directory: MSI: Retrieving a token from http://localhost:50342/oauth2/token, with payload {'resource': 'https://management.core.windows.net/'}
cli.azure.cli.core.auth.adal_authentication: MSIAuthenticationWrapper.get_token: scopes=('https://management.core.windows.net//.default',), kwargs={}
urllib3.connectionpool: Starting new HTTP connection (1): localhost:50342
urllib3.connectionpool: http://localhost:50342 "POST /oauth2/token HTTP/1.1" 200 3390
msrestazure.azure_active_directory: MSI: Retrieving a token from http://localhost:50342/oauth2/token, with payload {'resource': 'https://management.core.windows.net/'}
cli.azure.cli.core.auth.adal_authentication: Normalize expires_on: '1666748260' -> 1666748260
cli.azure.cli.core.sdk.policies: Request method: 'GET'
cli.azure.cli.core.sdk.policies: Request headers:
cli.azure.cli.core.sdk.policies:     'Accept': 'application/json'
cli.azure.cli.core.sdk.policies:     'x-ms-client-request-id': '1abc1aa0-54cb-11ed-bfa0-4a3932178544'
cli.azure.cli.core.sdk.policies:     'CommandName': 'staticwebapp update'
cli.azure.cli.core.sdk.policies:     'ParameterSetName': '-n -b -s -t --debug'
cli.azure.cli.core.sdk.policies:     'User-Agent': 'AZURECLI/2.41.0 (RPM) azsdk-python-azure-mgmt-web/7.0.0 Python/3.9.14 (Linux-5.4.0-1094-azure-x86_64-with-glibc2.35) cloud-shell/1.0'
cli.azure.cli.core.sdk.policies:     'Authorization': '*****'
cli.azure.cli.core.sdk.policies: Request body:
cli.azure.cli.core.sdk.policies: This request has no body
urllib3.connectionpool: Starting new HTTPS connection (1): management.azure.com:443
cli.azure.cli.core.sdk.policies: Response status: 200
cli.azure.cli.core.sdk.policies: Response headers:
cli.azure.cli.core.sdk.policies:     'Cache-Control': 'no-cache'
cli.azure.cli.core.sdk.policies:     'Pragma': 'no-cache'
cli.azure.cli.core.sdk.policies:     'Content-Type': 'application/json; charset=utf-8'
cli.azure.cli.core.sdk.policies:     'Content-Encoding': 'gzip'
cli.azure.cli.core.sdk.policies:     'Expires': '-1'
cli.azure.cli.core.sdk.policies:     'Vary': 'Accept-Encoding'
cli.azure.cli.core.sdk.policies:     'x-ms-original-request-ids': '6424da25-4feb-4916-94b4-739c96067215, df1ba5bf-d8da-4787-ab99-999caadd2637'
cli.azure.cli.core.sdk.policies:     'x-ms-ratelimit-remaining-subscription-reads': '11994'
cli.azure.cli.core.sdk.policies:     'x-ms-request-id': '858b241b-1e26-4ddd-b63f-0f0aa243193c'
cli.azure.cli.core.sdk.policies:     'x-ms-correlation-request-id': '858b241b-1e26-4ddd-b63f-0f0aa243193c'
cli.azure.cli.core.sdk.policies:     'x-ms-routing-request-id': 'SOUTHEASTASIA:20221026T011124Z:858b241b-1e26-4ddd-b63f-0f0aa243193c'
cli.azure.cli.core.sdk.policies:     'Strict-Transport-Security': 'max-age=31536000; includeSubDomains'
cli.azure.cli.core.sdk.policies:     'X-Content-Type-Options': 'nosniff'
cli.azure.cli.core.sdk.policies:     'Date': 'Wed, 26 Oct 2022 01:11:24 GMT'
cli.azure.cli.core.sdk.policies:     'Content-Length': '907'
cli.azure.cli.core.sdk.policies: Response content:
[],"privateEndpointConnections":[],"stagingEnvironmentPolicy":"Enabled","allowConfigFileUpdates":true,"contentDistributionEndpoint":"https://content-dm1.infrastructure.2.azurestaticapps.net","keyVaultReferenceIdentity":"SystemAssigned","userProvidedFunctionApps":null,"linkedBackends":[],"provider":"GitHub","enterpriseGradeCdnStatus":"Disabled","publicNetworkAccess":null},"sku":[],"stagingEnvironmentPolicy":"Enabled","allowConfigFileUpdates":true,"contentDistributionEndpoint":"https://content-dm1.infrastructure.2.azurestaticapps.net","keyVaultReferenceIdentity":"SystemAssigned","userProvidedFunctionApps":null,"linkedBackends":[],"provider":"None","enterpriseGradeCdnStatus":"Disabled","publicNetworkAccess":null},"sku":[],"stagingEnvironmentPolicy":"Enabled","allowConfigFileUpdates":true,"contentDistributionEndpoint":"https://content-hk1.infrastructure.2.azurestaticapps.net","keyVaultReferenceIdentity":"SystemAssigned","userProvidedFunctionApps":null,"linkedBackends":[],"provider":"None","enterpriseGradeCdnStatus":"Disabled","publicNetworkAccess":null},"sku":[],"stagingEnvironmentPolicy":"Enabled","allowConfigFileUpdates":true,"contentDistributionEndpoint":"https://content-hk1.infrastructure.2.azurestaticapps.net","keyVaultReferenceIdentity":"SystemAssigned","userProvidedFunctionApps":null,"linkedBackends":[],"provider":"None","enterpriseGradeCdnStatus":"Disabled","publicNetworkAccess":null},"sku":{"name":"Standard","tier":"Standard"}}]}
cli.azure.cli.core.sdk.policies: Request method: 'GET'
cli.azure.cli.core.sdk.policies: Request headers:
cli.azure.cli.core.sdk.policies:     'Accept': 'application/json'
cli.azure.cli.core.sdk.policies:     'x-ms-client-request-id': '1abc1aa0-54cb-11ed-bfa0-4a3932178544'
cli.azure.cli.core.sdk.policies:     'CommandName': 'staticwebapp update'
cli.azure.cli.core.sdk.policies:     'ParameterSetName': '-n -b -s -t --debug'
cli.azure.cli.core.sdk.policies:     'User-Agent': 'AZURECLI/2.41.0 (RPM) azsdk-python-azure-mgmt-web/7.0.0 Python/3.9.14 (Linux-5.4.0-1094-azure-x86_64-with-glibc2.35) cloud-shell/1.0'
cli.azure.cli.core.sdk.policies:     'Authorization': '*****'
cli.azure.cli.core.sdk.policies: Request body:
cli.azure.cli.core.sdk.policies: This request has no body
200 None
cli.azure.cli.core.sdk.policies: Response status: 200
cli.azure.cli.core.sdk.policies: Response headers:
cli.azure.cli.core.sdk.policies:     'Cache-Control': 'no-cache'
cli.azure.cli.core.sdk.policies:     'Pragma': 'no-cache'
cli.azure.cli.core.sdk.policies:     'Transfer-Encoding': 'chunked'
cli.azure.cli.core.sdk.policies:     'Content-Type': 'application/json'
cli.azure.cli.core.sdk.policies:     'Content-Encoding': 'gzip'
cli.azure.cli.core.sdk.policies:     'Expires': '-1'
cli.azure.cli.core.sdk.policies:     'Vary': 'Accept-Encoding'
cli.azure.cli.core.sdk.policies:     'Strict-Transport-Security': 'max-age=31536000; includeSubDomains'
cli.azure.cli.core.sdk.policies:     'x-ms-request-id': '442706dd-929b-4a22-a0a5-25e1d1ee68e9'
cli.azure.cli.core.sdk.policies:     'Server': 'Microsoft-IIS/10.0'
cli.azure.cli.core.sdk.policies:     'X-AspNet-Version': '4.0.30319'
cli.azure.cli.core.sdk.policies:     'X-Powered-By': 'ASP.NET'
cli.azure.cli.core.sdk.policies:     'x-ms-ratelimit-remaining-subscription-reads': '11993'
cli.azure.cli.core.sdk.policies:     'x-ms-correlation-request-id': 'ca5ec8ee-83ef-4345-bd31-83c594d1015a'
cli.azure.cli.core.sdk.policies:     'x-ms-routing-request-id': 'SOUTHEASTASIA:20221026T011124Z:ca5ec8ee-83ef-4345-bd31-83c594d1015a'
cli.azure.cli.core.sdk.policies:     'X-Content-Type-Options': 'nosniff'
cli.azure.cli.core.sdk.policies:     'Date': 'Wed, 26 Oct 2022 01:11:24 GMT'
cli.azure.cli.core.sdk.policies: Response content:
{"defaultHostname":"victorious-hill-0ecda7410.2.azurestaticapps.net","repositoryUrl":"https://github.com/cocoojbs/static","branch":"main","customDomains":[],"privateEndpointConnections":[],"stagingEnvironmentPolicy":"Enabled","allowConfigFileUpdates":true,"contentDistributionEndpoint":"https://content-dm1.infrastructure.2.azurestaticapps.net","keyVaultReferenceIdentity":"SystemAssigned","userProvidedFunctionApps":null,"linkedBackends":[],"provider":"GitHub","enterpriseGradeCdnStatus":"Disabled","publicNetworkAccess":null},"sku":{"name":"Free","tier":"Free"}}
cli.azure.cli.core.commands.client_factory: Getting management service client client_type=WebSiteManagementClient
urllib3.connectionpool: Starting new HTTP connection (1): localhost:50342
urllib3.connectionpool: http://localhost:50342 "POST /oauth2/token HTTP/1.1" 200 3390
msrestazure.azure_active_directory: MSI: Retrieving a token from http://localhost:50342/oauth2/token, with payload {'resource': 'https://management.core.windows.net/'}
cli.azure.cli.core.auth.adal_authentication: MSIAuthenticationWrapper.get_token: scopes=('https://management.core.windows.net//.default',), kwargs={}
urllib3.connectionpool: Starting new HTTP connection (1): localhost:50342
urllib3.connectionpool: http://localhost:50342 "POST /oauth2/token HTTP/1.1" 200 3390
msrestazure.azure_active_directory: MSI: Retrieving a token from http://localhost:50342/oauth2/token, with payload {'resource': 'https://management.core.windows.net/'}
cli.azure.cli.core.auth.adal_authentication: Normalize expires_on: '1666748260' -> 1666748260
cli.azure.cli.core.sdk.policies: Request method: 'GET'
cli.azure.cli.core.sdk.policies: Request headers:
cli.azure.cli.core.sdk.policies:     'Accept': 'application/json'
cli.azure.cli.core.sdk.policies:     'x-ms-client-request-id': '1abc1aa0-54cb-11ed-bfa0-4a3932178544'
cli.azure.cli.core.sdk.policies:     'CommandName': 'staticwebapp update'
cli.azure.cli.core.sdk.policies:     'ParameterSetName': '-n -b -s -t --debug'
cli.azure.cli.core.sdk.policies:     'User-Agent': 'AZURECLI/2.41.0 (RPM) azsdk-python-azure-mgmt-web/7.0.0 Python/3.9.14 (Linux-5.4.0-1094-azure-x86_64-with-glibc2.35) cloud-shell/1.0'
cli.azure.cli.core.sdk.policies:     'Authorization': '*****'
cli.azure.cli.core.sdk.policies: Request body:
cli.azure.cli.core.sdk.policies: This request has no body
urllib3.connectionpool: Starting new HTTPS connection (1): management.azure.com:443
cli.azure.cli.core.sdk.policies: Response status: 200
cli.azure.cli.core.sdk.policies: Response headers:
cli.azure.cli.core.sdk.policies:     'Cache-Control': 'no-cache'
cli.azure.cli.core.sdk.policies:     'Pragma': 'no-cache'
cli.azure.cli.core.sdk.policies:     'Content-Type': 'application/json; charset=utf-8'
cli.azure.cli.core.sdk.policies:     'Content-Encoding': 'gzip'
cli.azure.cli.core.sdk.policies:     'Expires': '-1'
cli.azure.cli.core.sdk.policies:     'Vary': 'Accept-Encoding'
cli.azure.cli.core.sdk.policies:     'x-ms-original-request-ids': 'a8a5cfc6-219c-46a9-b276-22e237389615, 4adce546-ec61-46c8-b6d4-866c608cf3b8'
cli.azure.cli.core.sdk.policies:     'x-ms-ratelimit-remaining-subscription-reads': '11998'
cli.azure.cli.core.sdk.policies:     'x-ms-request-id': '9700f009-e5a8-45b8-a921-0d3d88062b15'
cli.azure.cli.core.sdk.policies:     'x-ms-correlation-request-id': '9700f009-e5a8-45b8-a921-0d3d88062b15'
cli.azure.cli.core.sdk.policies:     'x-ms-routing-request-id': 'SOUTHEASTASIA:20221026T011124Z:9700f009-e5a8-45b8-a921-0d3d88062b15'
cli.azure.cli.core.sdk.policies:     'Strict-Transport-Security': 'max-age=31536000; includeSubDomains'
cli.azure.cli.core.sdk.policies:     'X-Content-Type-Options': 'nosniff'
cli.azure.cli.core.sdk.policies:     'Date': 'Wed, 26 Oct 2022 01:11:24 GMT'
cli.azure.cli.core.sdk.policies:     'Content-Length': '907'
cli.azure.cli.core.sdk.policies: Response content:
{"defaultHostname":"victorious-hill-0ecda7410.2.azurestaticapps.net","repositoryUrl":"https://github.com/cocoojbs/static","branch":"main","customDomains":[],"privateEndpointConnections":[],"stagingEnvironmentPolicy":"Enabled","allowConfigFileUpdates":true,"contentDistributionEndpoint":"https://content-dm1.infrastructure.2.azurestaticapps.net","keyVaultReferenceIdentity":"SystemAssigned","userProvidedFunctionApps":null,"linkedBackends":[],"provider":"GitHub","enterpriseGradeCdnStatus":"Disabled","publicNetworkAccess":null},"sku":{[],"stagingEnvironmentPolicy":"Enabled","allowConfigFileUpdates":true,"contentDistributionEndpoint":"https://content-dm1.infrastructure.2.azurestaticapps.net","keyVaultReferenceIdentity":"SystemAssigned","userProvidedFunctionApps":null,"linkedBackends":[],"provider":"None","enterpriseGradeCdnStatus":"Disabled","publicNetworkAccess":null},"sku":{[],"stagingEnvironmentPolicy":"Enabled","allowConfigFileUpdates":true,"contentDistributionEndpoint":"https://content-hk1.infrastructure.2.azurestaticapps.net","keyVaultReferenceIdentity":"SystemAssigned","userProvidedFunctionApps":null,"linkedBackends":[],"provider":"None","enterpriseGradeCdnStatus":"Disabled","publicNetworkAccess":null},"sku":[],"stagingEnvironmentPolicy":"Enabled","allowConfigFileUpdates":true,"contentDistributionEndpoint":"https://content-hk1.infrastructure.2.azurestaticapps.net","keyVaultReferenceIdentity":"SystemAssigned","userProvidedFunctionApps":null,"linkedBackends":[],"provider":"None","enterpriseGradeCdnStatus":"Disabled","publicNetworkAccess":null},"sku":cli.azure.cli.core.sdk.policies: Request method: 'PATCH'
cli.azure.cli.core.sdk.policies: Request headers:
cli.azure.cli.core.sdk.policies:     'Content-Type': 'application/json'
cli.azure.cli.core.sdk.policies:     'Content-Length': '263'
cli.azure.cli.core.sdk.policies:     'Accept': 'application/json'
cli.azure.cli.core.sdk.policies:     'x-ms-client-request-id': '1abc1aa0-54cb-11ed-bfa0-4a3932178544'
cli.azure.cli.core.sdk.policies:     'CommandName': 'staticwebapp update'
cli.azure.cli.core.sdk.policies:     'ParameterSetName': '-n -b -s -t --debug'
cli.azure.cli.core.sdk.policies:     'User-Agent': 'AZURECLI/2.41.0 (RPM) azsdk-python-azure-mgmt-web/7.0.0 Python/3.9.14 (Linux-5.4.0-1094-azure-x86_64-with-glibc2.35) cloud-shell/1.0'
cli.azure.cli.core.sdk.policies:     'Authorization': '*****'
cli.azure.cli.core.sdk.policies: Request body:
200 None
cli.azure.cli.core.sdk.policies: Response status: 200
cli.azure.cli.core.sdk.policies: Response headers:
cli.azure.cli.core.sdk.policies:     'Cache-Control': 'no-cache'
cli.azure.cli.core.sdk.policies:     'Pragma': 'no-cache'
cli.azure.cli.core.sdk.policies:     'Transfer-Encoding': 'chunked'
cli.azure.cli.core.sdk.policies:     'Content-Type': 'application/json'
cli.azure.cli.core.sdk.policies:     'Content-Encoding': 'gzip'
cli.azure.cli.core.sdk.policies:     'Expires': '-1'
cli.azure.cli.core.sdk.policies:     'Vary': 'Accept-Encoding'
cli.azure.cli.core.sdk.policies:     'Strict-Transport-Security': 'max-age=31536000; includeSubDomains'
cli.azure.cli.core.sdk.policies:     'x-ms-request-id': 'd4890f9c-518e-49a1-a3c5-abca72bfb7d6'
cli.azure.cli.core.sdk.policies:     'Server': 'Microsoft-IIS/10.0'
cli.azure.cli.core.sdk.policies:     'X-AspNet-Version': '4.0.30319'
cli.azure.cli.core.sdk.policies:     'X-Powered-By': 'ASP.NET'
cli.azure.cli.core.sdk.policies:     'x-ms-ratelimit-remaining-subscription-writes': '1199'
cli.azure.cli.core.sdk.policies:     'x-ms-correlation-request-id': '36270809-b60e-4337-a30b-5f99de648de7'
cli.azure.cli.core.sdk.policies:     'x-ms-routing-request-id': 'SOUTHEASTASIA:20221026T011127Z:36270809-b60e-4337-a30b-5f99de648de7'
cli.azure.cli.core.sdk.policies:     'X-Content-Type-Options': 'nosniff'
cli.azure.cli.core.sdk.policies:     'Date': 'Wed, 26 Oct 2022 01:11:26 GMT'
cli.azure.cli.core.sdk.policies: Response content:
{"defaultHostname":"victorious-hill-0ecda7410.2.azurestaticapps.net","repositoryUrl":"https://github.com/cocoojbs/public","branch":"main","customDomains":[],"privateEndpointConnections":[],"stagingEnvironmentPolicy":"Enabled","allowConfigFileUpdates":true,"contentDistributionEndpoint":"https://content-dm1.infrastructure.2.azurestaticapps.net","keyVaultReferenceIdentity":"SystemAssigned","userProvidedFunctionApps":null,"linkedBackends":[],"provider":"GitHub","enterpriseGradeCdnStatus":"Disabled","publicNetworkAccess":null},"sku":{"name":"Free","tier":"Free"}}
cli.knack.cli: Event: CommandInvoker.OnTransformResult [<function _resource_group_transform at 0x7f1b22bf1af0>, <function _x509_from_base64_to_hex_transform at 0x7f1b22bf1b80>]
cli.knack.cli: Event: CommandInvoker.OnFilterResult []
{
  "allowConfigFileUpdates": true,
  "branch": "main",
  "buildProperties": null,
  "contentDistributionEndpoint": "https://content-dm1.infrastructure.2.azurestaticapps.net",
  "customDomains": [],
  "defaultHostname": "victorious-hill-0ecda7410.2.azurestaticapps.net",
  "enterpriseGradeCdnStatus": "Disabled",
  "identity": null,
  "keyVaultReferenceIdentity": "SystemAssigned",
  "kind": null,
  "linkedBackends": [],
  "location": "Central US",
  "name": "kitamurat-webapp",
  "privateEndpointConnections": [],
  "provider": "GitHub",
  "publicNetworkAccess": null,
  "repositoryToken": null,
  "repositoryUrl": "https://github.com/cocoojbs/public",   #it has changed!
  "resourceGroup": "kitamurat-rg",
  "sku": {
    "capabilities": null,
    "capacity": null,
    "family": null,
    "locations": null,
    "name": "Free",
    "size": null,
    "skuCapacity": null,
    "tier": "Free"
  },
  "stagingEnvironmentPolicy": "Enabled",
  "tags": {
    "Owner": "kitamurat@jbs.com"
  },
  "templateProperties": null,
  "type": "Microsoft.Web/staticSites",
  "userProvidedFunctionApps": null
}
cli.knack.cli: Event: Cli.SuccessfulExecute []
cli.knack.cli: Event: Cli.PostExecute [<function AzCliLogging.deinit_cmd_metadata_logging at 0x7f1b22c25dc0>]
az_command_data_logger: exit code: 0
cli.__main__: Command ran in 5.382 seconds (init: 0.132, invoke: 5.250)
telemetry.main: Begin splitting cli events and extra events, total events: 1
telemetry.client: Accumulated 0 events. Flush the clients.
telemetry.main: Finish splitting cli events and extra events, cli events: 1
telemetry.save: Save telemetry record of length 3163 in cache
telemetry.check: Returns Positive.
telemetry.main: Begin creating telemetry upload process.
telemetry.process: Creating upload process: "/usr/bin/python3.9 /usr/lib/az/lib/python3.9/site-packages/azure/cli/telemetry/__init__.py /home/kitamurat/.azure"
telemetry.process: Return from creating process
telemetry.main: Finish creating telemetry upload process.

**Comment by PramodValavala-MSFT** [comment]
@cocoojbs The command would have created a workflow flow in the GitHub repository which upon successfully running, should deploy your application to static web apps. This might take a few minutes and you can track the workflow status in the GitHub repo actions tab.

**Comment by cocoojbs** [comment]
@PramodValavala-MSFT 
I was able to do what I wanted to do: after running the Update command, I configured the "Deploy web app to Azure Static Web Apps" workflow from the Github repository. Deploy web app to Azure Static Web Apps" from the Github repository workflow, and the repository switch was successful. (I had to add a StaticWebapps deployment token to the repository secret to configure the workflow).

Thank you very much for your support!
